docs/openapi.yaml

```diff
@@ -138,7 +138,7 @@ paths:
       - Authorization
       summary: Request an OAuth2 token
       description: |
-        Request an OAuth2 token from an authorization code.
+        Request an OAuth2 token from an authorization code "Client credential grant type".
```

Comment:
Do you think that the grant type should be 'authorization_code' since the system will use the authorization_code to exchange token from auth server?

Reply:
Sorry, I think I had confused client credential grant type with password grant type before.